experiment/resources/runner-startup-script-template.sh

```diff
@@ -43,4 +43,5 @@ docker run \
 -e LOCAL_EXPERIMENT={{local_experiment}} \
 {{additional_env}} {% if not local_experiment %}--name=runner-container {% endif %}\
 --cap-add SYS_NICE --cap-add SYS_PTRACE \
+--name fuzzbench-{{instance_name}} \
```

Comment:
Let's get rid of the `if not local_experiment` part on line 44.
Also drop `fuzzbench-` prefix please.

Reply:
Could you give some reasons to delete that prefix?

I think, to distinguish other docker containers in local machines, `fuzzbench-` should preserve.
For example, I also have some machine learning projects and run some docker containers.
Adding a prefix is more convenient to distinguish and, also more importantly, prevent delete other non-fuzzbench docker containers.

Reply:
I think it's unlikely someone would have other containers on their machine named `r-$EXPERIMENT-$TRIAL_ID`, and I'd rather not have this so that we don't have to deal with complexity, we can just use the instance naming regex we use in stop_experiment.py to stop the containers as well.

Reply:
I'm fine with this.